src/lib/backend.ai-client-esm.ts

```diff
@@ -673,6 +673,9 @@ class Client {
       this._features['deprecated-max-vfolder-count-in-keypair-resource-policy'] = true;
       this._features['deprecated-max-vfolder-size-in-keypair-resource-policy'] = true;
     }
+    if (this.isManagerVersionCompatibleWith('23.09.5')) {
+      this._features['use-win-instead-of-win32'] = true;
+    }
```

Comment:
I think it's supposed to use win32 from version 23.09.5 onwards, but this part seems to work in reverse.